server/superdesk/publish/odbc_test.py

```diff
@@ -15,7 +17,7 @@
 
 
 class ODBCTests(TestCase):
-    subscribers = [{"_id": "1", "name": "Test", "can_send_takes_packages": False, "media_type": "media",
+    subscribers = [{"_id": "1", "name": "Test", "subscriber_type": SUBSCRIBER_TYPES.WIRE, "media_type": "media",
```

Comment:
Not sure. Need to check with Andrew Lasky and Vince. But at the moment it's not playing any role in Superdesk.